biocontainers/Dockerfile

```diff
@@ -9,7 +9,8 @@ LABEL software.version="08252016"
 LABEL about.summary="Base image for BioDocker"
```

Comment:
as a base image, shouldn't it follow the same rules as other containers ie having:
biocontainers/x.y.z/Dockerfile

to keep track of older versions. Automatic builds won't be able to detect version from directory